DEBUG 2019-03-25 14:47:23,825 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/login.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'None': None, 'False': False}, {'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fa386fbf630>, 'request': <WSGIRequest: GET '/general_elections/admin/login/?next=/general_elections/admin/'>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fa386f82ba8>, 'user': <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x7fa386fbf668>>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fa386fe4ea0>>, 'DEFAULT_MESSAGE_LEVELS': {'INFO': 20, 'ERROR': 40, 'SUCCESS': 25, 'DEBUG': 10, 'WARNING': 30}, 'MEDIA_URL': '/elections2019/static/images/'}, {}, {'next': '/general_elections/admin/', 'LANGUAGE_BIDI': False, 'has_permission': False, 'LANGUAGE_CODE': 'en-us', 'view': <django.contrib.auth.views.LoginView object at 0x7fa386fbf7b8>, 'app_path': '/general_elections/admin/login/?next=/general_elections/admin/', 'site_name': 'swc.iitg.ac.in', 'site_header': 'Django administration', 'title': 'Log in', 'username': '', 'available_apps': [], 'site': <django.contrib.sites.requests.RequestSite object at 0x7fa386fbfa20>, 'form': <AdminAuthenticationForm bound=False, valid=Unknown, fields=(username;password)>, 'site_url': '/', 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 14:47:23,831 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/login.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'None': None, 'False': False}, {'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fa386fbf630>, 'request': <WSGIRequest: GET '/general_elections/admin/login/?next=/general_elections/admin/'>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fa386f82ba8>, 'user': <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x7fa386fbf668>>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fa386fe4ea0>>, 'DEFAULT_MESSAGE_LEVELS': {'INFO': 20, 'ERROR': 40, 'SUCCESS': 25, 'DEBUG': 10, 'WARNING': 30}, 'MEDIA_URL': '/elections2019/static/images/'}, {}, {'next': '/general_elections/admin/', 'LANGUAGE_BIDI': False, 'has_permission': False, 'LANGUAGE_CODE': 'en-us', 'view': <django.contrib.auth.views.LoginView object at 0x7fa386fbf7b8>, 'app_path': '/general_elections/admin/login/?next=/general_elections/admin/', 'site_name': 'swc.iitg.ac.in', 'site_header': 'Django administration', 'title': 'Log in', 'username': '', 'available_apps': [], 'site': <django.contrib.sites.requests.RequestSite object at 0x7fa386fbfa20>, 'form': <AdminAuthenticationForm bound=False, valid=Unknown, fields=(username;password)>, 'site_url': '/', 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 14:47:45,663 django.db.backends.execute:111 - (0.124) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."username" = 'elections2019'; args=('elections2019',)
DEBUG 2019-03-25 14:47:45,797 django.db.backends.execute:111 - (0.021) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = 'q6c2vy422se3v5z2ten9ch7un50xfsjg' LIMIT 1; args=('q6c2vy422se3v5z2ten9ch7un50xfsjg',)
DEBUG 2019-03-25 14:47:45,808 django.db.backends.execute:111 - (0.010) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") VALUES ('q6c2vy422se3v5z2ten9ch7un50xfsjg', 'ZWU4ZjYyZDc3YzJhYzFiODVhNWRkODFkYThlNTExYWViMmQwNjc5Zjp7fQ==', '2019-04-08T14:47:45.797917+00:00'::timestamptz); args=('q6c2vy422se3v5z2ten9ch7un50xfsjg', 'ZWU4ZjYyZDc3YzJhYzFiODVhNWRkODFkYThlNTExYWViMmQwNjc5Zjp7fQ==', datetime.datetime(2019, 4, 8, 14, 47, 45, 797917, tzinfo=<UTC>))
DEBUG 2019-03-25 14:47:45,818 django.db.backends.execute:111 - (0.007) UPDATE "auth_user" SET "last_login" = '2019-03-25T14:47:45.809048+00:00'::timestamptz WHERE "auth_user"."id" = 1; args=(datetime.datetime(2019, 3, 25, 14, 47, 45, 809048, tzinfo=<UTC>), 1)
DEBUG 2019-03-25 14:47:45,819 django.db.backends.execute:111 - (0.000) UPDATE "django_session" SET "session_data" = 'N2U0YjhiNThjM2NjNGRjZGRmMWI3ZDdhMGRkOTA0ZjI3ZTU3NDhlZTp7Il9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIiwiX2F1dGhfdXNlcl9oYXNoIjoiZjEzM2E0ODY4MGViNWRjOTYwZjYyYTljN2M0OTY0ZDQzODA4MzY3ZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', "expire_date" = '2019-04-08T14:47:45.819304+00:00'::timestamptz WHERE "django_session"."session_key" = 'q6c2vy422se3v5z2ten9ch7un50xfsjg'; args=('N2U0YjhiNThjM2NjNGRjZGRmMWI3ZDdhMGRkOTA0ZjI3ZTU3NDhlZTp7Il9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIiwiX2F1dGhfdXNlcl9oYXNoIjoiZjEzM2E0ODY4MGViNWRjOTYwZjYyYTljN2M0OTY0ZDQzODA4MzY3ZCIsIl9hdXRoX3VzZXJfaWQiOiIxIn0=', datetime.datetime(2019, 4, 8, 14, 47, 45, 819304, tzinfo=<UTC>), 'q6c2vy422se3v5z2ten9ch7un50xfsjg')
DEBUG 2019-03-25 14:47:45,855 django.db.backends.execute:111 - (0.007) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."session_key" = 'q6c2vy422se3v5z2ten9ch7un50xfsjg' AND "django_session"."expire_date" > '2019-03-25T14:47:45.838458+00:00'::timestamptz); args=('q6c2vy422se3v5z2ten9ch7un50xfsjg', datetime.datetime(2019, 3, 25, 14, 47, 45, 838458, tzinfo=<UTC>))
DEBUG 2019-03-25 14:47:45,856 django.db.backends.execute:111 - (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."id" = 1; args=(1,)
DEBUG 2019-03-25 14:47:45,880 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'None': None, 'False': False}, {'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fa38705e080>, 'request': <WSGIRequest: GET '/general_elections/admin/'>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fa386f9e550>, 'user': <SimpleLazyObject: <User: elections2019>>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fa386f44ea0>>, 'DEFAULT_MESSAGE_LEVELS': {'INFO': 20, 'ERROR': 40, 'SUCCESS': 25, 'DEBUG': 10, 'WARNING': 30}, 'MEDIA_URL': '/elections2019/static/images/'}, {}, {'LANGUAGE_BIDI': False, 'site_header': 'Django administration', 'title': 'Site administration', 'LANGUAGE_CODE': 'en-us', 'available_apps': [{'app_label': 'auth', 'models': [{'add_url': '/general_elections/admin/auth/group/add/', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Group', 'name': 'Groups'}, {'add_url': '/general_elections/admin/auth/user/add/', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'User', 'name': 'Users'}], 'app_url': '/general_elections/admin/auth/', 'has_module_perms': True, 'name': 'Authentication and Authorization'}, {'app_label': 'general_elections', 'models': [{'add_url': '/general_elections/admin/general_elections/admin/add/', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Admin', 'name': 'Admins'}, {'add_url': '/general_elections/admin/general_elections/contestants/add/', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Contestants', 'name': 'Contestantss'}, {'add_url': '/general_elections/admin/general_elections/volunteer/add/', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Volunteer', 'name': 'Volunteers'}, {'add_url': '/general_elections/admin/general_elections/voterlist/add/', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'VoterList', 'name': 'Voter lists'}], 'app_url': '/general_elections/admin/general_elections/', 'has_module_perms': True, 'name': 'General_Elections'}], 'app_list': [{'app_label': 'auth', 'models': [{'add_url': '/general_elections/admin/auth/group/add/', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Group', 'name': 'Groups'}, {'add_url': '/general_elections/admin/auth/user/add/', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'User', 'name': 'Users'}], 'app_url': '/general_elections/admin/auth/', 'has_module_perms': True, 'name': 'Authentication and Authorization'}, {'app_label': 'general_elections', 'models': [{'add_url': '/general_elections/admin/general_elections/admin/add/', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Admin', 'name': 'Admins'}, {'add_url': '/general_elections/admin/general_elections/contestants/add/', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Contestants', 'name': 'Contestantss'}, {'add_url': '/general_elections/admin/general_elections/volunteer/add/', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Volunteer', 'name': 'Volunteers'}, {'add_url': '/general_elections/admin/general_elections/voterlist/add/', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'VoterList', 'name': 'Voter lists'}], 'app_url': '/general_elections/admin/general_elections/', 'has_module_perms': True, 'name': 'General_Elections'}], 'has_permission': True, 'site_url': '/', 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 14:47:45,881 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'None': None, 'False': False}, {'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fa38705e080>, 'request': <WSGIRequest: GET '/general_elections/admin/'>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fa386f9e550>, 'user': <SimpleLazyObject: <User: elections2019>>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fa386f44ea0>>, 'DEFAULT_MESSAGE_LEVELS': {'INFO': 20, 'ERROR': 40, 'SUCCESS': 25, 'DEBUG': 10, 'WARNING': 30}, 'MEDIA_URL': '/elections2019/static/images/'}, {}, {'LANGUAGE_BIDI': False, 'site_header': 'Django administration', 'title': 'Site administration', 'LANGUAGE_CODE': 'en-us', 'available_apps': [{'app_label': 'auth', 'models': [{'add_url': '/general_elections/admin/auth/group/add/', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Group', 'name': 'Groups'}, {'add_url': '/general_elections/admin/auth/user/add/', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'User', 'name': 'Users'}], 'app_url': '/general_elections/admin/auth/', 'has_module_perms': True, 'name': 'Authentication and Authorization'}, {'app_label': 'general_elections', 'models': [{'add_url': '/general_elections/admin/general_elections/admin/add/', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Admin', 'name': 'Admins'}, {'add_url': '/general_elections/admin/general_elections/contestants/add/', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Contestants', 'name': 'Contestantss'}, {'add_url': '/general_elections/admin/general_elections/volunteer/add/', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Volunteer', 'name': 'Volunteers'}, {'add_url': '/general_elections/admin/general_elections/voterlist/add/', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'VoterList', 'name': 'Voter lists'}], 'app_url': '/general_elections/admin/general_elections/', 'has_module_perms': True, 'name': 'General_Elections'}], 'app_list': [{'app_label': 'auth', 'models': [{'add_url': '/general_elections/admin/auth/group/add/', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Group', 'name': 'Groups'}, {'add_url': '/general_elections/admin/auth/user/add/', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'User', 'name': 'Users'}], 'app_url': '/general_elections/admin/auth/', 'has_module_perms': True, 'name': 'Authentication and Authorization'}, {'app_label': 'general_elections', 'models': [{'add_url': '/general_elections/admin/general_elections/admin/add/', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Admin', 'name': 'Admins'}, {'add_url': '/general_elections/admin/general_elections/contestants/add/', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Contestants', 'name': 'Contestantss'}, {'add_url': '/general_elections/admin/general_elections/volunteer/add/', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Volunteer', 'name': 'Volunteers'}, {'add_url': '/general_elections/admin/general_elections/voterlist/add/', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'VoterList', 'name': 'Voter lists'}], 'app_url': '/general_elections/admin/general_elections/', 'has_module_perms': True, 'name': 'General_Elections'}], 'has_permission': True, 'site_url': '/', 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 14:47:45,930 django.db.backends.execute:111 - (0.041) SELECT "django_admin_log"."id", "django_admin_log"."action_time", "django_admin_log"."user_id", "django_admin_log"."content_type_id", "django_admin_log"."object_id", "django_admin_log"."object_repr", "django_admin_log"."action_flag", "django_admin_log"."change_message", "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined", "django_content_type"."id", "django_content_type"."app_label", "django_content_type"."model" FROM "django_admin_log" INNER JOIN "auth_user" ON ("django_admin_log"."user_id" = "auth_user"."id") LEFT OUTER JOIN "django_content_type" ON ("django_admin_log"."content_type_id" = "django_content_type"."id") WHERE "django_admin_log"."user_id" = 1 ORDER BY "django_admin_log"."action_time" DESC LIMIT 10; args=(1,)
DEBUG 2019-03-25 14:47:53,135 django.db.backends.execute:111 - (0.002) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."session_key" = 'q6c2vy422se3v5z2ten9ch7un50xfsjg' AND "django_session"."expire_date" > '2019-03-25T14:47:53.123861+00:00'::timestamptz); args=('q6c2vy422se3v5z2ten9ch7un50xfsjg', datetime.datetime(2019, 3, 25, 14, 47, 53, 123861, tzinfo=<UTC>))
DEBUG 2019-03-25 14:47:53,137 django.db.backends.execute:111 - (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."id" = 1; args=(1,)
DEBUG 2019-03-25 14:47:53,141 django.db.backends.execute:111 - (0.001) SELECT (1) AS "a" FROM "general_elections_admin" WHERE "general_elections_admin"."user_id" = 1 LIMIT 1; args=(1,)
DEBUG 2019-03-25 14:47:57,534 django.db.backends.execute:111 - (0.002) SELECT "general_elections_contestants"."webmail_id", "general_elections_contestants"."name", "general_elections_contestants"."agenda1", "general_elections_contestants"."agenda2", "general_elections_contestants"."agenda3", "general_elections_contestants"."image", "general_elections_contestants"."post", "general_elections_contestants"."mobile_no", "general_elections_contestants"."eligible", "general_elections_contestants"."comments" FROM "general_elections_contestants" ORDER BY "general_elections_contestants"."post" ASC; args=()
DEBUG 2019-03-25 14:48:05,814 django.db.backends.execute:111 - (0.002) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."session_key" = 'q6c2vy422se3v5z2ten9ch7un50xfsjg' AND "django_session"."expire_date" > '2019-03-25T14:48:05.801150+00:00'::timestamptz); args=('q6c2vy422se3v5z2ten9ch7un50xfsjg', datetime.datetime(2019, 3, 25, 14, 48, 5, 801150, tzinfo=<UTC>))
DEBUG 2019-03-25 14:48:05,816 django.db.backends.execute:111 - (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."id" = 1; args=(1,)
DEBUG 2019-03-25 14:48:05,819 django.db.backends.execute:111 - (0.000) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE "django_session"."session_key" = 'q6c2vy422se3v5z2ten9ch7un50xfsjg'; args=('q6c2vy422se3v5z2ten9ch7un50xfsjg',)
DEBUG 2019-03-25 14:48:05,820 django.db.backends.execute:111 - (0.000) DELETE FROM "django_session" WHERE "django_session"."session_key" IN ('q6c2vy422se3v5z2ten9ch7un50xfsjg'); args=('q6c2vy422se3v5z2ten9ch7un50xfsjg',)
DEBUG 2019-03-25 14:48:05,834 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'registration/logged_out.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'None': None, 'False': False}, {'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fa386f9acc0>, 'request': <WSGIRequest: GET '/general_elections/admin/logout/'>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fa386fa5160>, 'user': <django.contrib.auth.models.AnonymousUser object at 0x7fa386f57dd8>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fa386f44c80>>, 'DEFAULT_MESSAGE_LEVELS': {'INFO': 20, 'ERROR': 40, 'SUCCESS': 25, 'DEBUG': 10, 'WARNING': 30}, 'MEDIA_URL': '/elections2019/static/images/'}, {}, {'LANGUAGE_BIDI': False, 'available_apps': [{'app_label': 'auth', 'models': [{'add_url': '/general_elections/admin/auth/group/add/', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Group', 'name': 'Groups'}, {'add_url': '/general_elections/admin/auth/user/add/', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'User', 'name': 'Users'}], 'app_url': '/general_elections/admin/auth/', 'has_module_perms': True, 'name': 'Authentication and Authorization'}, {'app_label': 'general_elections', 'models': [{'add_url': '/general_elections/admin/general_elections/admin/add/', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Admin', 'name': 'Admins'}, {'add_url': '/general_elections/admin/general_elections/contestants/add/', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Contestants', 'name': 'Contestantss'}, {'add_url': '/general_elections/admin/general_elections/volunteer/add/', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Volunteer', 'name': 'Volunteers'}, {'add_url': '/general_elections/admin/general_elections/voterlist/add/', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'VoterList', 'name': 'Voter lists'}], 'app_url': '/general_elections/admin/general_elections/', 'has_module_perms': True, 'name': 'General_Elections'}], 'LANGUAGE_CODE': 'en-us', 'view': <django.contrib.auth.views.LogoutView object at 0x7fa386f57cf8>, 'site_name': 'swc.iitg.ac.in', 'site_header': 'Django administration', 'title': 'Logged out', 'has_permission': False, 'site': <django.contrib.sites.requests.RequestSite object at 0x7fa386f57b00>, 'site_url': '/', 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 14:48:05,835 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'registration/logged_out.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'None': None, 'False': False}, {'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fa386f9acc0>, 'request': <WSGIRequest: GET '/general_elections/admin/logout/'>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fa386fa5160>, 'user': <django.contrib.auth.models.AnonymousUser object at 0x7fa386f57dd8>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fa386f44c80>>, 'DEFAULT_MESSAGE_LEVELS': {'INFO': 20, 'ERROR': 40, 'SUCCESS': 25, 'DEBUG': 10, 'WARNING': 30}, 'MEDIA_URL': '/elections2019/static/images/'}, {}, {'LANGUAGE_BIDI': False, 'available_apps': [{'app_label': 'auth', 'models': [{'add_url': '/general_elections/admin/auth/group/add/', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Group', 'name': 'Groups'}, {'add_url': '/general_elections/admin/auth/user/add/', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'User', 'name': 'Users'}], 'app_url': '/general_elections/admin/auth/', 'has_module_perms': True, 'name': 'Authentication and Authorization'}, {'app_label': 'general_elections', 'models': [{'add_url': '/general_elections/admin/general_elections/admin/add/', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Admin', 'name': 'Admins'}, {'add_url': '/general_elections/admin/general_elections/contestants/add/', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Contestants', 'name': 'Contestantss'}, {'add_url': '/general_elections/admin/general_elections/volunteer/add/', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'Volunteer', 'name': 'Volunteers'}, {'add_url': '/general_elections/admin/general_elections/voterlist/add/', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'object_name': 'VoterList', 'name': 'Voter lists'}], 'app_url': '/general_elections/admin/general_elections/', 'has_module_perms': True, 'name': 'General_Elections'}], 'LANGUAGE_CODE': 'en-us', 'view': <django.contrib.auth.views.LogoutView object at 0x7fa386f57cf8>, 'site_name': 'swc.iitg.ac.in', 'site_header': 'Django administration', 'title': 'Logged out', 'has_permission': False, 'site': <django.contrib.sites.requests.RequestSite object at 0x7fa386f57b00>, 'site_url': '/', 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 14:48:07,763 django.db.backends.execute:111 - (0.001) SELECT "general_elections_contestants"."webmail_id", "general_elections_contestants"."name", "general_elections_contestants"."agenda1", "general_elections_contestants"."agenda2", "general_elections_contestants"."agenda3", "general_elections_contestants"."image", "general_elections_contestants"."post", "general_elections_contestants"."mobile_no", "general_elections_contestants"."eligible", "general_elections_contestants"."comments" FROM "general_elections_contestants" ORDER BY "general_elections_contestants"."post" ASC; args=()
DEBUG 2019-03-25 14:48:13,740 django.db.backends.execute:111 - (0.001) SELECT "general_elections_contestants"."webmail_id", "general_elections_contestants"."name", "general_elections_contestants"."agenda1", "general_elections_contestants"."agenda2", "general_elections_contestants"."agenda3", "general_elections_contestants"."image", "general_elections_contestants"."post", "general_elections_contestants"."mobile_no", "general_elections_contestants"."eligible", "general_elections_contestants"."comments" FROM "general_elections_contestants" ORDER BY "general_elections_contestants"."post" ASC; args=()
DEBUG 2019-03-25 14:48:14,428 django.db.backends.execute:111 - (0.001) SELECT "general_elections_contestants"."webmail_id", "general_elections_contestants"."name", "general_elections_contestants"."agenda1", "general_elections_contestants"."agenda2", "general_elections_contestants"."agenda3", "general_elections_contestants"."image", "general_elections_contestants"."post", "general_elections_contestants"."mobile_no", "general_elections_contestants"."eligible", "general_elections_contestants"."comments" FROM "general_elections_contestants" ORDER BY "general_elections_contestants"."post" ASC; args=()
DEBUG 2019-03-25 14:48:14,947 django.db.backends.execute:111 - (0.001) SELECT "general_elections_contestants"."webmail_id", "general_elections_contestants"."name", "general_elections_contestants"."agenda1", "general_elections_contestants"."agenda2", "general_elections_contestants"."agenda3", "general_elections_contestants"."image", "general_elections_contestants"."post", "general_elections_contestants"."mobile_no", "general_elections_contestants"."eligible", "general_elections_contestants"."comments" FROM "general_elections_contestants" ORDER BY "general_elections_contestants"."post" ASC; args=()
DEBUG 2019-03-25 14:48:15,185 django.db.backends.execute:111 - (0.001) SELECT "general_elections_contestants"."webmail_id", "general_elections_contestants"."name", "general_elections_contestants"."agenda1", "general_elections_contestants"."agenda2", "general_elections_contestants"."agenda3", "general_elections_contestants"."image", "general_elections_contestants"."post", "general_elections_contestants"."mobile_no", "general_elections_contestants"."eligible", "general_elections_contestants"."comments" FROM "general_elections_contestants" ORDER BY "general_elections_contestants"."post" ASC; args=()
DEBUG 2019-03-25 14:48:15,388 django.db.backends.execute:111 - (0.001) SELECT "general_elections_contestants"."webmail_id", "general_elections_contestants"."name", "general_elections_contestants"."agenda1", "general_elections_contestants"."agenda2", "general_elections_contestants"."agenda3", "general_elections_contestants"."image", "general_elections_contestants"."post", "general_elections_contestants"."mobile_no", "general_elections_contestants"."eligible", "general_elections_contestants"."comments" FROM "general_elections_contestants" ORDER BY "general_elections_contestants"."post" ASC; args=()
DEBUG 2019-03-25 14:48:15,601 django.db.backends.execute:111 - (0.001) SELECT "general_elections_contestants"."webmail_id", "general_elections_contestants"."name", "general_elections_contestants"."agenda1", "general_elections_contestants"."agenda2", "general_elections_contestants"."agenda3", "general_elections_contestants"."image", "general_elections_contestants"."post", "general_elections_contestants"."mobile_no", "general_elections_contestants"."eligible", "general_elections_contestants"."comments" FROM "general_elections_contestants" ORDER BY "general_elections_contestants"."post" ASC; args=()
DEBUG 2019-03-25 14:48:16,097 django.db.backends.execute:111 - (0.001) SELECT "general_elections_contestants"."webmail_id", "general_elections_contestants"."name", "general_elections_contestants"."agenda1", "general_elections_contestants"."agenda2", "general_elections_contestants"."agenda3", "general_elections_contestants"."image", "general_elections_contestants"."post", "general_elections_contestants"."mobile_no", "general_elections_contestants"."eligible", "general_elections_contestants"."comments" FROM "general_elections_contestants" ORDER BY "general_elections_contestants"."post" ASC; args=()
ERROR 2019-03-25 14:48:33,669 django.request.handle_uncaught_exception:118 - Internal Server Error: /general_elections/admin_login/
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/webapps/elections2019/IITG_General_Elections/general_elections/views/views_admin.py", line 142, in admin_login_new
    username = request.session.get('mail', None).split('@')[0]
AttributeError: 'NoneType' object has no attribute 'split'
ERROR 2019-03-25 15:07:08,677 django.request.handle_uncaught_exception:118 - Internal Server Error: /general_elections/
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/core/handlers/base.py", line 113, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/urls/resolvers.py", line 494, in resolve
    for pattern in self.url_patterns:
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/utils/functional.py", line 36, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/urls/resolvers.py", line 536, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/utils/functional.py", line 36, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/urls/resolvers.py", line 529, in urlconf_module
    return import_module(self.urlconf_name)
  File "/webapps/elections2019/venv/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/webapps/elections2019/IITG_General_Elections/voting/urls.py", line 23, in <module>
    url('general_elections/authentication/', include('authentication.urls', namespace='authentication')),
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/webapps/elections2019/venv/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/webapps/elections2019/IITG_General_Elections/authentication/urls.py", line 2, in <module>
    from authentication import views
  File "/webapps/elections2019/IITG_General_Elections/authentication/views.py", line 24
    redirect_uri = request.build_absolute_uri(reverse('authentication:gettoken'))
    ^
IndentationError: unexpected indent
ERROR 2019-03-25 16:35:18,479 django.request.handle_uncaught_exception:118 - Internal Server Error: /general_elections/admin_login/
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/webapps/elections2019/IITG_General_Elections/general_elections/views/views_admin.py", line 142, in admin_login_new
    username = request.session.get('mail', None).split('@')[0]
AttributeError: 'NoneType' object has no attribute 'split'
DEBUG 2019-03-25 16:35:26,853 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/login.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'False': False, 'None': None}, {'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fb745449048>>, 'MEDIA_URL': '/elections2019/static/images/', 'request': <WSGIRequest: GET '/general_elections/admin/login/?next=/general_elections/admin/'>, 'user': <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x7fb745de2320>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fb7454355f8>, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fb745de22b0>, 'DEFAULT_MESSAGE_LEVELS': {'WARNING': 30, 'ERROR': 40, 'INFO': 20, 'SUCCESS': 25, 'DEBUG': 10}}, {}, {'has_permission': False, 'next': '/general_elections/admin/', 'site_header': 'Django administration', 'LANGUAGE_CODE': 'en-us', 'form': <AdminAuthenticationForm bound=False, valid=Unknown, fields=(username;password)>, 'site_url': '/', 'username': '', 'title': 'Log in', 'site_name': 'swc.iitg.ac.in', 'app_path': '/general_elections/admin/login/?next=/general_elections/admin/', 'LANGUAGE_BIDI': False, 'view': <django.contrib.auth.views.LoginView object at 0x7fb7454a6d30>, 'site': <django.contrib.sites.requests.RequestSite object at 0x7fb7454542e8>, 'available_apps': [], 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 16:35:26,854 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/login.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'False': False, 'None': None}, {'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fb745449048>>, 'MEDIA_URL': '/elections2019/static/images/', 'request': <WSGIRequest: GET '/general_elections/admin/login/?next=/general_elections/admin/'>, 'user': <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x7fb745de2320>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fb7454355f8>, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fb745de22b0>, 'DEFAULT_MESSAGE_LEVELS': {'WARNING': 30, 'ERROR': 40, 'INFO': 20, 'SUCCESS': 25, 'DEBUG': 10}}, {}, {'has_permission': False, 'next': '/general_elections/admin/', 'site_header': 'Django administration', 'LANGUAGE_CODE': 'en-us', 'form': <AdminAuthenticationForm bound=False, valid=Unknown, fields=(username;password)>, 'site_url': '/', 'username': '', 'title': 'Log in', 'site_name': 'swc.iitg.ac.in', 'app_path': '/general_elections/admin/login/?next=/general_elections/admin/', 'LANGUAGE_BIDI': False, 'view': <django.contrib.auth.views.LoginView object at 0x7fb7454a6d30>, 'site': <django.contrib.sites.requests.RequestSite object at 0x7fb7454542e8>, 'available_apps': [], 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 16:35:38,677 django.db.backends.execute:111 - (0.002) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."username" = 'elections2019'; args=('elections2019',)
DEBUG 2019-03-25 16:35:38,816 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/login.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'False': False, 'None': None}, {'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fb7453d7ea0>>, 'MEDIA_URL': '/elections2019/static/images/', 'request': <WSGIRequest: POST '/general_elections/admin/login/?next=/general_elections/admin/'>, 'user': <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x7fb745458390>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fb745435c18>, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fb745458320>, 'DEFAULT_MESSAGE_LEVELS': {'WARNING': 30, 'ERROR': 40, 'INFO': 20, 'SUCCESS': 25, 'DEBUG': 10}}, {}, {'has_permission': False, 'next': '/general_elections/admin/', 'site_header': 'Django administration', 'LANGUAGE_CODE': 'en-us', 'form': <AdminAuthenticationForm bound=True, valid=False, fields=(username;password)>, 'site_url': '/', 'username': '', 'title': 'Log in', 'site_name': 'swc.iitg.ac.in', 'app_path': '/general_elections/admin/login/?next=/general_elections/admin/', 'LANGUAGE_BIDI': False, 'view': <django.contrib.auth.views.LoginView object at 0x7fb745458940>, 'site': <django.contrib.sites.requests.RequestSite object at 0x7fb7a5d39eb8>, 'available_apps': [], 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 16:35:38,816 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/login.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'False': False, 'None': None}, {'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fb7453d7ea0>>, 'MEDIA_URL': '/elections2019/static/images/', 'request': <WSGIRequest: POST '/general_elections/admin/login/?next=/general_elections/admin/'>, 'user': <SimpleLazyObject: <django.contrib.auth.models.AnonymousUser object at 0x7fb745458390>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fb745435c18>, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fb745458320>, 'DEFAULT_MESSAGE_LEVELS': {'WARNING': 30, 'ERROR': 40, 'INFO': 20, 'SUCCESS': 25, 'DEBUG': 10}}, {}, {'has_permission': False, 'next': '/general_elections/admin/', 'site_header': 'Django administration', 'LANGUAGE_CODE': 'en-us', 'form': <AdminAuthenticationForm bound=True, valid=False, fields=(username;password)>, 'site_url': '/', 'username': '', 'title': 'Log in', 'site_name': 'swc.iitg.ac.in', 'app_path': '/general_elections/admin/login/?next=/general_elections/admin/', 'LANGUAGE_BIDI': False, 'view': <django.contrib.auth.views.LoginView object at 0x7fb745458940>, 'site': <django.contrib.sites.requests.RequestSite object at 0x7fb7a5d39eb8>, 'available_apps': [], 'site_title': 'Django site admin'}]
DEBUG 2019-03-25 16:35:47,259 django.db.backends.execute:111 - (0.002) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."username" = 'elections2019'; args=('elections2019',)
DEBUG 2019-03-25 16:35:47,362 django.db.backends.execute:111 - (0.001) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = '8gmznsc4vsiyqn6upkwla1437gs1q52d' LIMIT 1; args=('8gmznsc4vsiyqn6upkwla1437gs1q52d',)
DEBUG 2019-03-25 16:35:47,363 django.db.backends.execute:111 - (0.001) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") VALUES ('8gmznsc4vsiyqn6upkwla1437gs1q52d', 'ZWU4ZjYyZDc3YzJhYzFiODVhNWRkODFkYThlNTExYWViMmQwNjc5Zjp7fQ==', '2019-04-08T16:35:47.362910+00:00'::timestamptz); args=('8gmznsc4vsiyqn6upkwla1437gs1q52d', 'ZWU4ZjYyZDc3YzJhYzFiODVhNWRkODFkYThlNTExYWViMmQwNjc5Zjp7fQ==', datetime.datetime(2019, 4, 8, 16, 35, 47, 362910, tzinfo=<UTC>))
DEBUG 2019-03-25 16:35:47,365 django.db.backends.execute:111 - (0.001) UPDATE "auth_user" SET "last_login" = '2019-03-25T16:35:47.364449+00:00'::timestamptz WHERE "auth_user"."id" = 1; args=(datetime.datetime(2019, 3, 25, 16, 35, 47, 364449, tzinfo=<UTC>), 1)
DEBUG 2019-03-25 16:35:47,367 django.db.backends.execute:111 - (0.000) UPDATE "django_session" SET "session_data" = 'N2E4NjMyOTQ0Y2VmZmY5N2NhNWZlYmVmZGE1ZTJlN2U2MTQ3MjZhNDp7Il9hdXRoX3VzZXJfaGFzaCI6ImYxMzNhNDg2ODBlYjVkYzk2MGY2MmE5YzdjNDk2NGQ0MzgwODM2N2QiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', "expire_date" = '2019-04-08T16:35:47.366388+00:00'::timestamptz WHERE "django_session"."session_key" = '8gmznsc4vsiyqn6upkwla1437gs1q52d'; args=('N2E4NjMyOTQ0Y2VmZmY5N2NhNWZlYmVmZGE1ZTJlN2U2MTQ3MjZhNDp7Il9hdXRoX3VzZXJfaGFzaCI6ImYxMzNhNDg2ODBlYjVkYzk2MGY2MmE5YzdjNDk2NGQ0MzgwODM2N2QiLCJfYXV0aF91c2VyX2lkIjoiMSIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', datetime.datetime(2019, 4, 8, 16, 35, 47, 366388, tzinfo=<UTC>), '8gmznsc4vsiyqn6upkwla1437gs1q52d')
DEBUG 2019-03-25 16:35:47,563 django.db.backends.execute:111 - (0.002) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2019-03-25T16:35:47.551172+00:00'::timestamptz AND "django_session"."session_key" = '8gmznsc4vsiyqn6upkwla1437gs1q52d'); args=(datetime.datetime(2019, 3, 25, 16, 35, 47, 551172, tzinfo=<UTC>), '8gmznsc4vsiyqn6upkwla1437gs1q52d')
DEBUG 2019-03-25 16:35:47,565 django.db.backends.execute:111 - (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."id" = 1; args=(1,)
DEBUG 2019-03-25 16:35:47,627 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'False': False, 'None': None}, {'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fb7453b4ae8>>, 'MEDIA_URL': '/elections2019/static/images/', 'request': <WSGIRequest: GET '/general_elections/admin/'>, 'user': <SimpleLazyObject: <User: elections2019>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fb7453d2ef0>, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fb7454cbe80>, 'DEFAULT_MESSAGE_LEVELS': {'WARNING': 30, 'ERROR': 40, 'INFO': 20, 'SUCCESS': 25, 'DEBUG': 10}}, {}, {'has_permission': True, 'site_header': 'Django administration', 'LANGUAGE_CODE': 'en-us', 'site_title': 'Django site admin', 'LANGUAGE_BIDI': False, 'site_url': '/', 'title': 'Site administration', 'available_apps': [{'has_module_perms': True, 'name': 'Authentication and Authorization', 'app_label': 'auth', 'models': [{'name': 'Groups', 'object_name': 'Group', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/auth/group/add/'}, {'name': 'Users', 'object_name': 'User', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/auth/user/add/'}], 'app_url': '/general_elections/admin/auth/'}, {'has_module_perms': True, 'name': 'General_Elections', 'app_label': 'general_elections', 'models': [{'name': 'Admins', 'object_name': 'Admin', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/admin/add/'}, {'name': 'Contestantss', 'object_name': 'Contestants', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/contestants/add/'}, {'name': 'Volunteers', 'object_name': 'Volunteer', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/volunteer/add/'}, {'name': 'Voter lists', 'object_name': 'VoterList', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/voterlist/add/'}], 'app_url': '/general_elections/admin/general_elections/'}], 'app_list': [{'has_module_perms': True, 'name': 'Authentication and Authorization', 'app_label': 'auth', 'models': [{'name': 'Groups', 'object_name': 'Group', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/auth/group/add/'}, {'name': 'Users', 'object_name': 'User', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/auth/user/add/'}], 'app_url': '/general_elections/admin/auth/'}, {'has_module_perms': True, 'name': 'General_Elections', 'app_label': 'general_elections', 'models': [{'name': 'Admins', 'object_name': 'Admin', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/admin/add/'}, {'name': 'Contestantss', 'object_name': 'Contestants', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/contestants/add/'}, {'name': 'Volunteers', 'object_name': 'Volunteer', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/volunteer/add/'}, {'name': 'Voter lists', 'object_name': 'VoterList', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/voterlist/add/'}], 'app_url': '/general_elections/admin/general_elections/'}]}]
DEBUG 2019-03-25 16:35:47,629 django.template._resolve_lookup:878 - Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in [{'True': True, 'False': False, 'None': None}, {'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fb7453b4ae8>>, 'MEDIA_URL': '/elections2019/static/images/', 'request': <WSGIRequest: GET '/general_elections/admin/'>, 'user': <SimpleLazyObject: <User: elections2019>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fb7453d2ef0>, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fb7454cbe80>, 'DEFAULT_MESSAGE_LEVELS': {'WARNING': 30, 'ERROR': 40, 'INFO': 20, 'SUCCESS': 25, 'DEBUG': 10}}, {}, {'has_permission': True, 'site_header': 'Django administration', 'LANGUAGE_CODE': 'en-us', 'site_title': 'Django site admin', 'LANGUAGE_BIDI': False, 'site_url': '/', 'title': 'Site administration', 'available_apps': [{'has_module_perms': True, 'name': 'Authentication and Authorization', 'app_label': 'auth', 'models': [{'name': 'Groups', 'object_name': 'Group', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/auth/group/add/'}, {'name': 'Users', 'object_name': 'User', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/auth/user/add/'}], 'app_url': '/general_elections/admin/auth/'}, {'has_module_perms': True, 'name': 'General_Elections', 'app_label': 'general_elections', 'models': [{'name': 'Admins', 'object_name': 'Admin', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/admin/add/'}, {'name': 'Contestantss', 'object_name': 'Contestants', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/contestants/add/'}, {'name': 'Volunteers', 'object_name': 'Volunteer', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/volunteer/add/'}, {'name': 'Voter lists', 'object_name': 'VoterList', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/voterlist/add/'}], 'app_url': '/general_elections/admin/general_elections/'}], 'app_list': [{'has_module_perms': True, 'name': 'Authentication and Authorization', 'app_label': 'auth', 'models': [{'name': 'Groups', 'object_name': 'Group', 'admin_url': '/general_elections/admin/auth/group/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/auth/group/add/'}, {'name': 'Users', 'object_name': 'User', 'admin_url': '/general_elections/admin/auth/user/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/auth/user/add/'}], 'app_url': '/general_elections/admin/auth/'}, {'has_module_perms': True, 'name': 'General_Elections', 'app_label': 'general_elections', 'models': [{'name': 'Admins', 'object_name': 'Admin', 'admin_url': '/general_elections/admin/general_elections/admin/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/admin/add/'}, {'name': 'Contestantss', 'object_name': 'Contestants', 'admin_url': '/general_elections/admin/general_elections/contestants/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/contestants/add/'}, {'name': 'Volunteers', 'object_name': 'Volunteer', 'admin_url': '/general_elections/admin/general_elections/volunteer/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/volunteer/add/'}, {'name': 'Voter lists', 'object_name': 'VoterList', 'admin_url': '/general_elections/admin/general_elections/voterlist/', 'perms': {'change': True, 'delete': True, 'add': True}, 'add_url': '/general_elections/admin/general_elections/voterlist/add/'}], 'app_url': '/general_elections/admin/general_elections/'}]}]
DEBUG 2019-03-25 16:35:47,635 django.db.backends.execute:111 - (0.002) SELECT "django_admin_log"."id", "django_admin_log"."action_time", "django_admin_log"."user_id", "django_admin_log"."content_type_id", "django_admin_log"."object_id", "django_admin_log"."object_repr", "django_admin_log"."action_flag", "django_admin_log"."change_message", "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined", "django_content_type"."id", "django_content_type"."app_label", "django_content_type"."model" FROM "django_admin_log" INNER JOIN "auth_user" ON ("django_admin_log"."user_id" = "auth_user"."id") LEFT OUTER JOIN "django_content_type" ON ("django_admin_log"."content_type_id" = "django_content_type"."id") WHERE "django_admin_log"."user_id" = 1 ORDER BY "django_admin_log"."action_time" DESC LIMIT 10; args=(1,)
DEBUG 2019-03-25 16:35:50,612 django.db.backends.execute:111 - (0.002) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2019-03-25T16:35:50.594042+00:00'::timestamptz AND "django_session"."session_key" = '8gmznsc4vsiyqn6upkwla1437gs1q52d'); args=(datetime.datetime(2019, 3, 25, 16, 35, 50, 594042, tzinfo=<UTC>), '8gmznsc4vsiyqn6upkwla1437gs1q52d')
DEBUG 2019-03-25 16:35:55,134 django.db.backends.execute:111 - (0.002) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2019-03-25T16:35:55.119910+00:00'::timestamptz AND "django_session"."session_key" = '8gmznsc4vsiyqn6upkwla1437gs1q52d'); args=(datetime.datetime(2019, 3, 25, 16, 35, 55, 119910, tzinfo=<UTC>), '8gmznsc4vsiyqn6upkwla1437gs1q52d')
DEBUG 2019-03-25 16:35:55,136 django.db.backends.execute:111 - (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."id" = 1; args=(1,)
DEBUG 2019-03-25 16:35:55,138 django.db.backends.execute:111 - (0.001) SELECT (1) AS "a" FROM "general_elections_admin" WHERE "general_elections_admin"."user_id" = 1 LIMIT 1; args=(1,)
DEBUG 2019-03-25 16:35:57,074 django.db.backends.execute:111 - (0.002) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2019-03-25T16:35:57.062140+00:00'::timestamptz AND "django_session"."session_key" = '8gmznsc4vsiyqn6upkwla1437gs1q52d'); args=(datetime.datetime(2019, 3, 25, 16, 35, 57, 62140, tzinfo=<UTC>), '8gmznsc4vsiyqn6upkwla1437gs1q52d')
DEBUG 2019-03-25 16:35:57,076 django.db.backends.execute:111 - (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."id" = 1; args=(1,)
DEBUG 2019-03-25 16:35:57,079 django.db.backends.execute:111 - (0.001) SELECT (1) AS "a" FROM "general_elections_admin" WHERE "general_elections_admin"."user_id" = 1 LIMIT 1; args=(1,)
DEBUG 2019-03-25 16:35:57,106 django.template._resolve_lookup:878 - Exception while resolving variable 'results_dict' in template 'results.html'.
Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 835, in _resolve_lookup
    current = current[bit]
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/context.py", line 83, in __getitem__
    raise KeyError(key)
KeyError: 'results_dict'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 841, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'results_dict'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'results_dict'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/webapps/elections2019/venv/lib/python3.5/site-packages/django/template/base.py", line 856, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [results_dict] in [{'True': True, 'False': False, 'None': None}, {'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x7fb74543f510>>, 'MEDIA_URL': '/elections2019/static/images/', 'request': <WSGIRequest: GET '/general_elections/results/'>, 'user': <SimpleLazyObject: <User: elections2019>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7fb745432d30>, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7fb7454cfc88>, 'DEFAULT_MESSAGE_LEVELS': {'WARNING': 30, 'ERROR': 40, 'INFO': 20, 'SUCCESS': 25, 'DEBUG': 10}}, {}, {'sign_in_url': 'https://login.microsoftonline.com/850aa78d-94e1-4bc6-9cf3-8c11b530701c/oauth2/v2.0/authorize?client_id=2597b493-ebd9-4260-927d-f08b1865cb77&prompt=login&response_type=code&scope=openid+offline_access+User.Read+Mail.Read&redirect_uri=http%3A%2F%2Fswc.iitg.ac.in%2Fgeneral_elections%2Fauthentication%2Fgettoken%2F', 'already_logged_in': []}]
DEBUG 2019-03-25 16:35:57,108 django.db.backends.execute:111 - (0.001) UPDATE "django_session" SET "session_data" = 'YjgzOTc0NDhjNTU1N2ZjZTU1NDg0ZWVjM2U1YzE3YmI2NGQ4Yjg1Yzp7Il9hdXRoX3VzZXJfaGFzaCI6ImYxMzNhNDg2ODBlYjVkYzk2MGY2MmE5YzdjNDk2NGQ0MzgwODM2N2QiLCJzYXZlX3VzZXIiOnRydWUsIl9hdXRoX3VzZXJfaWQiOiIxIiwicmVkaXJlY3RfdXJsIjoicmVzdWx0cyIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', "expire_date" = '2019-04-08T16:35:57.107619+00:00'::timestamptz WHERE "django_session"."session_key" = '8gmznsc4vsiyqn6upkwla1437gs1q52d'; args=('YjgzOTc0NDhjNTU1N2ZjZTU1NDg0ZWVjM2U1YzE3YmI2NGQ4Yjg1Yzp7Il9hdXRoX3VzZXJfaGFzaCI6ImYxMzNhNDg2ODBlYjVkYzk2MGY2MmE5YzdjNDk2NGQ0MzgwODM2N2QiLCJzYXZlX3VzZXIiOnRydWUsIl9hdXRoX3VzZXJfaWQiOiIxIiwicmVkaXJlY3RfdXJsIjoicmVzdWx0cyIsIl9hdXRoX3VzZXJfYmFja2VuZCI6ImRqYW5nby5jb250cmliLmF1dGguYmFja2VuZHMuTW9kZWxCYWNrZW5kIn0=', datetime.datetime(2019, 4, 8, 16, 35, 57, 107619, tzinfo=<UTC>), '8gmznsc4vsiyqn6upkwla1437gs1q52d')
